docs/source/tools/codegen.rst

```diff
@@ -7,37 +7,31 @@ DAML codegen
 Introduction
 ============
 
-You can use the DAML codegen to generate Java and Scala classes representing DAML contract templates. These classes incorporate all boilerplate code for constructing corresponding ledger :ref:`com.daml.ledger.api.v1.CreateCommand` and :ref:`com.daml.ledger.api.v1.ExerciseCommand`.
+You can use the DAML codegen to generate Java and Scala classes representing DAML contract templates.
```

Comment:
You start by only talking about Java and Scala but then when it comes to running the codegen you introduce JavaScript.